nbgrader/exchange/exchange.py

```diff
@@ -76,6 +76,18 @@ def _cache_default(self):
 
     coursedir = Instance(CourseDirectory, allow_none=True)
 
+    groupshared = Bool(
```

Comment:
I think this should probably be an attribute of CourseDirectory, and then accessed via `self.coursedir.groupshared`, so that the config value only needs to be set once.

Reply:
done, thanks